test/unittests/test_plugin.py

```diff
@@ -619,3 +622,24 @@ def test_brightness_increment_decrease(self):
                 mock_call.assert_called_with("turn_on", {"brightness_step_pct": -20})
                 fake_bulb.decrease_brightness(50)
                 mock_call.assert_called_with("turn_on", {"brightness_step_pct": -50})
+
+    def test_automation_turns_off_if_configured(self):
+        automation_connector = FakeConnector()
+        automation_connector.turn_off = Mock()
+        self.plugin.registered_devices.pop()
+        self.plugin.registered_devices.append(
+        self.plugin.device_types["automation"](
+            automation_connector,
+            "test_automation",
+            "mdi:automation",
+            "test_automation",
+            "on",
+            {"friendly_name": "Test Automation"},
+            "Living Room",
+            None,
+            True,
+        )
+        )
+        self.plugin.handle_turn_off(FakeMessage("ovos.phal.plugin.homeassistant.turn.off", {"device_id": "test_automation"}, None))
+        self.plugin.bus.once("ovos.phal.plugin.homeassistant.turn.off.response", Mock())
```

Comment:
Did you mean to test this callback or is this registered event just leftover? Doesn't appear to do anything as is